Util.java

```diff
@@ -0,0 +1,11 @@
+package com.ski.biathlons;
+
+import java.util.Objects;
+
+public class Util {
```

Comment:
Nu merita creata o clasa pentru o singura metoda care cotine o singura linie de cod. Calculul timupului final inclusiv interpretarea penalizarilor trebuiau incapsulate in clasa `Athlete`.